### PR TITLE
Show permission icons in Actions Transactions Details

### LINKED ISSUE #3348: Icon and padding issue on Action and Motion pages + tooltips copy
## Steps to reproduce

1. Create an action or a motion.
2. Look at the transaction meta to see if the permission icon appears or not.

## Expected behaviour

* The correct permission icon should appear in all cases.
* The tooltip copy should be correct.

## Actual behaviour

* The permission icon is not appearing for many action types on action pages. Actions currently missing the icon:
      - Payments
      - Transferring funds
      - New team creation
      - Editing a team
      - Changing permissions
      - Recovery mode

a58725e2-5be1-4218-a0d3-9252b25671bb

* For motions, none should have an icon so the padding should be removed. [Figma link for spec.](https://www.figma.com/file/CCgGsBzhX3BUENS1IvgP7N/Motions-%26-Disputes?node-id=4736%3A22)

edfe9c1a-4ffd-48aa-aee3-0ada23b7f4c5

* There are also some issues with the tooltip copy that should be fixed in this update. Review any tooltips of icons fixed during this update for correctness.

The description for Root permission does not read correctly. Please change to:
> "This permission allows you to modify colony-wide parameters, upgrade the colony, and manage permissions in the Root domain."

      
      
d0b95609-a861-4cc1-afa0-66468a2ab346

The description for Arbitration permission needs a description. It should b:
> "Arbitration allows you to resolve disputes, make state changes, and punish bad behavior."

      
      
a7ec61f7-4aae-4a7e-88c5-1262ddf69957

### PR DESCRIPTION
**Pull request by dit7ya**
## Description

Now correct permissions are shown in Transaction details for actions.

This PR also corrects the extra spacing before the time in case of motion transaction details.

It also shows the arbitration permission's correct tooltip message. 

And it also slightly corrects the root permission's tooltip message.

Resolves  #3348

### CODE REVIEW

File: src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx [hunk 1]
```diff
@@ -427,39 +429,45 @@ const ActionsPageEvent = ({
             />
           </div>
           <div className={styles.details}>
-            <div className={styles.roles}>
-              {eventName &&
-                EVENT_ROLES_MAP[eventName] &&
-                EVENT_ROLES_MAP[eventName].map((role, index) => {
+            {eventName && EVENT_ROLES_MAP[eventName] && (
+              <div className={styles.roles}>
+                {EVENT_ROLES_MAP[eventName].map((role, index) => {
```
Comment (rdig) [inline src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx:434]: Maybe write this as 
```suggestion
                {EVENT_ROLES_MAP[eventName]?.map((role, index) => {
```

It shouldn't be really necessary here, but it's a good practice to safeguard

File: src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx [hunk 2]
```diff
@@ -427,39 +429,45 @@ const ActionsPageEvent = ({
...
+                    If it is an award action, do not display the Arbitration label */
                   if (
-                    (!values?.isSmiteAction && role === ColonyRole.Root) ||
-                    (values?.isSmiteAction && role !== ColonyRole.Root)
+                    (values?.actionType ===
+                      ColonyActions.EmitDomainReputationPenalty &&
+                      role === ColonyRole.Root) ||
+                    (values?.actionType ===
+                      ColonyActions.EmitDomainReputationReward &&
+                      role === ColonyRole.Arbitration)
```
Comment (rdig) [inline src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx:443]: Is the `values.isSmiteAction` value still returned ?

If we're not using it anymore _(and it's not used elsewhere)_ please remove it

**Review by rdig — APPROVED** [review]
Looks good. I've left some notes, but nothing that's a deal-breaker.

I'm pre-approving this. :+1:

**Review by ArmandoGraterol — APPROVED** [review]
Looking great! I like your spin on the conditional in the `ActionsPageEvent`

**Review by arrenv — APPROVED** [review]
Nice one, looks good to go.

8e006acc-c310-4718-8210-1e019cc8478d

e1cfbfbe-50e1-45be-a9ac-776c35e788f2

File: src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx (hunk 2 above)
Comment (dit7ya) [reply src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx:443]: It is being used at various other places, I took a look.

**Pull request merged**